LICENSE

```diff
@@ -0,0 +1,15 @@
+MIT No Attribution
+
```

Comment:
No real reason, just to keep the LICENSE file similar to my other projects. Thanks

```suggestion
The MIT License (MIT)

Copyright (c) 2020 Ariel Mashraki
```

Reply:
No problem. I copied the file from another project.